### PR TITLE
Novia713 spanish translation

### PR DESCRIPTION
**Pull request by novia713**
vamos a por el 100% :D

### CODE REVIEW

File: config/translations/es/generate.profile.yml [hunk 1]
```diff
@@ -1,21 +1,21 @@
-description: 'Generar un perfil de instalación.'
-help: 'El comando <info>generate:profile</info> ayuda a generar un perfil de instalación.'
-welcome: 'Bienvenido al generador de perfil de instalación'
+description: 'Genera un perfil.'
+help: 'El comando <info>generate:profile</info> le ayuda a generar un nuevo perfil de instalación.'
+welcome: 'Welcome to the Drupal installation profile generator'
```
Comment (jmolivas) [inline config/translations/es/generate.profile.yml]: Bebería quedar como 
> Bienvenido al generador de perfil de instalación

File: config/translations/es/generate.module.yml [hunk 2]
```diff
@@ -22,11 +22,12 @@ questions:
     core: 'Introduzca la versión del core de Drupal'
     package: 'Introduzca el nombre del paquete'
     feature: 'Definir el módulo como una feature'
-    module-file: '¿Quiere generar un archivo .module?'
-    composer: '¿Quiere añadir un archivo composer.json a tu módulo?'
-    dependencies: '¿Te gustaría añadir dependencias al módulo?'
-    test: '¿Quiere generar una clase de unit test?'
-    controller: '¿Quiere generar un controlador por defecto?'
+<<<<<<< HEAD
```
Comment (jmolivas) [inline config/translations/es/generate.module.yml]: Remover esta línea

**Comment by novia713** [comment]
ups!
will look into this tomorrow
i'll manage some few strings left untranslated with the merge

**Comment by novia713** [comment]
little errors fixed! :+1:

**Comment by jmolivas** [comment]
@novia713 PR was merged. This will be included on next release, Thanks.

**Pull request merged**